app/Http/Controllers/API/TraderController.php

```diff
@@ -173,6 +179,20 @@ public function emailVoucherHistory(Request $request, Trader $trader)
         return response()->json(['message' => $response_text], 202);
     }
 
+    public function getMinMaxVoucherDates($vouchers) {
+        $sorted_vouchers = $vouchers->sortBy(function($voucher) {
+            return $voucher->paymentPendedOn->created_at->timestamp;
+        });
+
+        $max_date = $sorted_vouchers[0]->paymentPendedOn->created_at->format('d-m-Y');
```

Comment:
sortby is ascendant small->big. so is max_date and min_date the wrong way round here. alternativly, sortByDesc() does it the other way.